lib/filesharing/files_basics/lib/src/local_file/local_file_data.dart

```diff
@@ -34,7 +34,7 @@ class LocalFileData extends LocalFile {
 
   @override
   getFile() {
-    throw UnimplementedError();
+    return null;
```

Comment:
Returning `null` makes for me more sense (It's also used in the `_uploadToFirebase` method).